libcontainer/factory_linux.go

```diff
@@ -12,6 +12,8 @@ import (
 	securejoin "github.com/cyphar/filepath-securejoin"
 	"golang.org/x/sys/unix"
 
+	//nolint:revive // Enable cgroup manager to manage devices
+	_ "github.com/opencontainers/runc/libcontainer/cgroups/devices"
```

Comment:
I am slightly concerned that if we drop this by accident, it wouldn't be a compile-time error, but I guess since we have integration tests that would almost all fail if we do forget to include this, I guess it's not that big of an issue.

Reply:
We always have some device rules, so the resulting runc won't be able to start a container -- that's a kind of error that should be caught by our tests.

Reply:
Yeah, fair enough.